content/en/docs/How Tos/metrics.md

```diff
@@ -26,23 +30,45 @@ type FilterStats struct {
 }
 ```
 
-In this case `NumProcessedLines` must represent the **total** number of processed 
-lines since Baker started, and `NumFilteredLines` is the number of discarded 
-(or filtered) records. Due to historical reasons these fields have the word
+In this case `NumProcessedLines` should represent the **total** number of processed 
+lines since the filter creation, while `NumFilteredLines` is the number of discarded 
+(i.e filtered) records. Due to historical reasons these fields have the word
 _lines_ in them but they do mean the number of records.
 
-An important point is that `Stats` can be called from any goroutine so it must be
-safe for concurrent use by multiple goroutines. 
+Let's say our filter needs to perform HTTP requests in order to decide whether a record
```

Comment:
what do you think of adding a title for this section? Like "A practical example" or so